src/Aspire.Hosting.PostgreSQL/PostgresBuilderExtensions.cs

```diff
@@ -48,7 +69,8 @@ public static IResourceBuilder<PostgresServerResource> AddPostgres(this IDistrib
                       {
                           context.EnvironmentVariables[UserEnvVarName] = postgresServer.UserNameReference;
                           context.EnvironmentVariables[PasswordEnvVarName] = postgresServer.PasswordParameter;
-                      });
+                      })
+                      .WithAnnotation(new HealthCheckAnnotation(healthCheckName));
```

Comment:
We should have a WithHealthCheck method right?

Reply:
We should also be able to add multiple health checks. The 1:1 mapping between the health check name and the resource name seems like something we need to fix.

Reply:
I was thinking about adding `WithHealthCheck(...)`. I actually wanted to make it something like this:

```csharp
builder.WithHealthCheck(healthCheckBuilder => {
   healthCheckBuilder.AddBlahCheck(...);
});
```

... and avoid having to set a stringly name altogether (or at least make it auto-generated). But the health checks API surface isn't conducive to that. I'll add a `WithHealthChecks(...)` just so we have the API but its not doing that much for us.